src/tools/bin/pic2xdmf.py

```diff
@@ -1,608 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why did you remocve this program - shouldn't it still work fine with openPMD-api hdf5 output?

Reply:
No this tool depends on splash data.

Reply:
Okay